### ISSUE TITLE
Allow `image()` template function to generate an `<img>` tag

### ISSUE THREAD

**Issue by fluffy-critter** [fluffy-critter/Publ#76]
## Expected Behavior

`markdown`'s `<img>` generation should be moved into `Image.get_img_tag()` so that templates can use it too.

The template `image()` function should get an Image object instead of a rendition.

`Image.get_rendition()` should handle trhe static path mapping.

`get_image` should be aware of remote/static images and should return a `RemoteImage` thing that also provides `get_rendition`/`get_img_tag`.

Basically `Image` should become a base class and then what's currently called that should be `ContentImage` or something.
